source/_integrations/unifiprotect.markdown

```diff
@@ -213,6 +213,14 @@ Your main UniFi Protect NVR device also gets a number of diagnostics sensors tha
 * **Disk Health**: Each disk installed in your NVR will have a disk health sensor. These are simple good/bad sensors and the order is not promised to match the order in UniFi OS. Disk model number is provided as a state attribute though to help map sensor to disk.
 * **Utilization and Storage Sensors**: Several other sensors are also added for uptime, hardware utilization, and distribution details of the video on disk.
 
+## Ignoring Devices
+
+If you do not want an UniFi Protect device to appear in Home Assistant, you can "delete" the device from Home Assistant.
+
+![Delete Device](/images/integrations/unifiprotect/delete_device.png)
+
+Deleting the device will ignore the device, preventing it from being added back to Home Assistant again. If you change your mind later, there is an ignored devices configuration option with a comma-separated list of devices you have ignored.
```

Comment:
So, now I wonder, why is this a comma-separated list?

It doesn't sound very user-friendly to have that in a single text field.

I suggest this is changed to use e.g., a multi-select list, from which people can de-select items to be removed.

Reply:
Sure, I will get right on that as soon as there are docs on how to add a select field to config entry options.

Reply:
> Sure, I will get right on that as soon as there are docs on how to add a select field to config entry options. 

Everybody is happy to help to get things right, and you can help document it too 👍 
Selectors are documented in the user documentation and there are many examples to be found in the code base.

This should not be merged in this state.

Reply:
The feature is already merged. You can choose not to merge this PR if you want, but that means the documentation is not being added, which is what you blocked the original PR for. Or you can revert the PR that is merged and take the feature away completely. 

It is the core developers' responsibility to document core features properly. As you made the argument in the other PR, it is _your_ project. Not mine. Core failed to provide the guidelines to make me (or the reviewer) capable of realizing there was the option to use selectors for config entries. It was literally a one sentence mention in a blog post that has no RSS feed, mailing list, or any other way to follow to get updates on. The PR was approved and merged by your process and thus a failure in your process.

This feature was already accepted. Regardless of how it happened, reverting it now when it is not "broken" just proves my point that core devs are over controlling. The feature is there. What you are asking for is an enhancement to an existing feature. If you want to revert the feature, do not expect me to make it again.

Reply:
You never consider how you say anything to developers. You just post a fix without content. Or tell them they are wrong with no reasoning. It is very hostile and confrontational. A better way to approach this would be

> Oh, it looks like you are using a CSV. You could have used a selector (link to docs) instead to make this a list. It looks like it was already merged without anyone noticing this. I am going to merge this now, but could you make another PR to convert this to a selector and try to get it in before the final 2022.9.0 cut?

Instead of just say "no this is wrong, I am not merging it".

Reply:
Sorry, I'm going to say the same about you. You also take it hostile, instead of saying:

> Sure, I will get right on that as soon as there are docs on how to add a select field to config entry options.

You could also say: "I couldn't find anything about how to do that, could you point me to more information?"

Things work out both ways. And yes, considering your low acceptance of review comments in past PRs, I will block this PR and will revert the parent PR if needed.

There are multiple approaches possible in this case. One uses a dropdown that contains a list of checkable items. When a user unchecks items, remove the unchecked items from the list and store them. This can be done using a `cv.multi_select`; the HomeKit option flow might be a good example where you can find usage of it.

Another option is using the `SelectSelector` for the field (instead of `str`). When the `SelectSelectorConfig` is passed you could pass `multiple=True` and (in  case you'd like user to be able to enter additional ignores) set `custom_values=True`as well.

Being able to use our selectors in the flows, is fairly new, so not much usage around yet. But, some can be found. You can find `selectors.py` in our helpers.

../Frenck